login.html

```diff
@@ -1,48 +1,53 @@
 				<br>
+				<input type="password" placeholder="Password" autocomplete="off" name="password" required />
 				<br>
 				<button type ="submit" value="Login"> Login</button>
-				<h2>Submit new feedback</h2>
-				<h2>View feedback results</h2>
-				<img style = "width: 33%; display: inline-block;" src= "results.jpg" alt="Image not available">
-				<img style="width: 33%; display: inline-block;" src="new feedback.jpg" alt="Image not available" >
-				<input type ="text"  placeholder="Username" autocomplete="on" name ="uname" required/>
-				<input type="password" placeholder="Password" autocomplete="off" name="password" required />
-				<p style="width: 66%; display: inline-block;">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla quam velit, vulputate eu pharetra nec, mattis ac neque. Duis vulputate commodo lectus, ac blandit elit tincidunt id. Sed rhoncus, tortor sed eleifend tristique, tortor mauris molestie elit, et lacinia ipsum quam nec dui. Quisque nec mauris sit amet elit iaculis pretium sit amet quis magna. Aenean velit odio, elementum in tempus ut, vehicula eu diam. Pellentesque rhoncus aliquam mattis. Ut vulputate eros sed felis sodales nec vulputate justo hendrerit. Vivamus varius pretium ligula, a aliquam odio euismod sit amet. Quisque laoreet sem sit amet orci ullamcorper at ultricies metus viverra. Pellentesque arcu mauris, malesuada quis ornare accumsan, blandit sed diam.</p>
-				<p style="width: 66%; display: inline-block;">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla quam velit, vulputate eu pharetra nec, mattis ac neque. Duis vulputate commodo lectus, ac blandit elit tincidunt id. Sed rhoncus, tortor sed eleifend tristique, tortor mauris molestie elit, et lacinia ipsum quam nec dui. Quisque nec mauris sit amet elit iaculis pretium sit amet quis magna. Aenean velit odio, elementum in tempus ut, vehicula eu diam. Pellentesque rhoncus aliquam mattis. Ut vulputate eros sed felis sodales nec vulputate justo hendrerit. Vivamus varius pretium ligula, a aliquam odio euismod sit amet. Quisque laoreet sem sit amet orci ullamcorper at ultricies metus viverra. Pellentesque arcu mauris, malesuada quis ornare accumsan, blandit sed diam</p>
-			</article>
-			</article>
 			</div>
-			</h3>
-			<article >
-			<article>
-			<div class="loggin">
-			<h3>
-			Interview Feedback
-		</div>
-		</div><br>
 		</form>
-		<div style = "width: 100%" class="info-section" >
-		<div style = "width: 100%" class="info-section"">
-		<form action="Evaluation.html">
-		<p><small>Copyright@Sofvision 2017</small></p>
-	</footer>
-	<footer style="position: absolute; right: 0; left: 0; bottom: 0; width:100%; padding:1rem; height: 50px; background-color: #dfdfdf; text-align: center;">
-  margin: 0;
-  min-height: 100%;">
-  padding-bottom: 6rem;
- <!DOCTYPE html>
+		</div>
+
+		  <div id="body-wrapper">
+	<div style = "width: 100%" class="info-section" >
+		
+		<article>
```

Comment:
I seems a good idea for me because the structure is made of : title and content.